<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="0" total="8" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Suite C" duration-ms="104934" started-at="2016-05-24T13:19:07Z" finished-at="2016-05-24T13:20:51Z">
    <groups>
    </groups>
    <test name="Test 1" duration-ms="104934" started-at="2016-05-24T13:19:07Z" finished-at="2016-05-24T13:20:51Z">
      <class name="com.ctliselenium.framework.datadriven.suiteC.Test1">
        <test-method status="PASS" signature="initLogs()[pri:0, instance:com.ctliselenium.framework.datadriven.suiteC.Test1@1723a349]" name="initLogs" is-config="true" duration-ms="2" started-at="2016-05-24T13:19:07Z" finished-at="2016-05-24T13:19:07Z">
        </test-method> <!-- initLogs -->
        <test-method status="PASS" signature="test1(java.util.Hashtable)[pri:0, instance:com.ctliselenium.framework.datadriven.suiteC.Test1@1723a349]" name="test1" duration-ms="104606" started-at="2016-05-24T13:19:07Z" data-provider="SuiteCDataProvider" finished-at="2016-05-24T13:20:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Col3=C33, Col2=C22, Runmode=Y, Browser=Mozilla, Result=}]]>
              </value>
            </param>
          </params>
        </test-method> <!-- test1 -->
        <test-method status="SKIP" signature="test1(java.util.Hashtable)[pri:0, instance:com.ctliselenium.framework.datadriven.suiteC.Test1@1723a349]" name="test1" duration-ms="156" started-at="2016-05-24T13:20:51Z" data-provider="SuiteCDataProvider" finished-at="2016-05-24T13:20:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Col3=C34, Col2=C23, Runmode=N, Browser=C21, Result=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[skipping the test=test1inside the suite=SuiteC]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: skipping the test=test1inside the suite=SuiteC
	at com.ctliselenium.framework.datadriven.TestBase.validateRunmodes(TestBase.java:88)
	at com.ctliselenium.framework.datadriven.suiteC.Test1.test1(Test1.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:702)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:768)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1188)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1184)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1113)
	at org.testng.TestNG.run(TestNG.java:1025)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
        </test-method> <!-- test1 -->
        <test-method status="SKIP" signature="test1(java.util.Hashtable)[pri:0, instance:com.ctliselenium.framework.datadriven.suiteC.Test1@1723a349]" name="test1" duration-ms="79" started-at="2016-05-24T13:20:51Z" data-provider="SuiteCDataProvider" finished-at="2016-05-24T13:20:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Col3=C35, Col2=C24, Runmode=N, Browser=C31, Result=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[skipping the test=test1inside the suite=SuiteC]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: skipping the test=test1inside the suite=SuiteC
	at com.ctliselenium.framework.datadriven.TestBase.validateRunmodes(TestBase.java:88)
	at com.ctliselenium.framework.datadriven.suiteC.Test1.test1(Test1.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:702)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:768)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1188)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1184)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1113)
	at org.testng.TestNG.run(TestNG.java:1025)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
        </test-method> <!-- test1 -->
      </class> <!-- com.ctliselenium.framework.datadriven.suiteC.Test1 -->
    </test> <!-- Test 1 -->
  </suite> <!-- Suite C -->
  <suite name="Ordering suite" duration-ms="80795" started-at="2016-05-24T13:16:50Z" finished-at="2016-05-24T13:18:11Z">
    <groups>
    </groups>
    <test name="login Test" duration-ms="45775" started-at="2016-05-24T13:16:50Z" finished-at="2016-05-24T13:17:36Z">
      <class name="com.ctliselenium.framework.datadriven.ordering.LoginTest">
        <test-method status="PASS" signature="initLogs()[pri:0, instance:com.ctliselenium.framework.datadriven.ordering.LoginTest@26f3bbad]" name="initLogs" is-config="true" duration-ms="312" started-at="2016-05-24T13:16:50Z" finished-at="2016-05-24T13:16:51Z">
        </test-method> <!-- initLogs -->
        <test-method status="SKIP" signature="loginTest(java.util.Hashtable)[pri:0, instance:com.ctliselenium.framework.datadriven.ordering.LoginTest@26f3bbad]" name="loginTest" duration-ms="97" started-at="2016-05-24T13:16:53Z" data-provider="OrderingDataProvider" finished-at="2016-05-24T13:16:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=C22, Password=C33, Runmode=N, Expected_Result=SUCCESS, Browser=Mozilla, Result=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[skipping the test=loginTestinside the suite=OrderingSuite]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: skipping the test=loginTestinside the suite=OrderingSuite
	at com.ctliselenium.framework.datadriven.TestBase.validateRunmodes(TestBase.java:88)
	at com.ctliselenium.framework.datadriven.ordering.LoginTest.loginTest(LoginTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:702)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:768)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1188)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1184)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1113)
	at org.testng.TestNG.run(TestNG.java:1025)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
        </test-method> <!-- loginTest -->
        <test-method status="SKIP" signature="loginTest(java.util.Hashtable)[pri:0, instance:com.ctliselenium.framework.datadriven.ordering.LoginTest@26f3bbad]" name="loginTest" duration-ms="94" started-at="2016-05-24T13:16:53Z" data-provider="OrderingDataProvider" finished-at="2016-05-24T13:16:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=C23, Password=C34, Runmode=N, Expected_Result=FAILURE, Browser=Mozilla, Result=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[skipping the test=loginTestinside the suite=OrderingSuite]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: skipping the test=loginTestinside the suite=OrderingSuite
	at com.ctliselenium.framework.datadriven.TestBase.validateRunmodes(TestBase.java:88)
	at com.ctliselenium.framework.datadriven.ordering.LoginTest.loginTest(LoginTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:702)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:768)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1188)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1184)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1113)
	at org.testng.TestNG.run(TestNG.java:1025)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="loginTest(java.util.Hashtable)[pri:0, instance:com.ctliselenium.framework.datadriven.ordering.LoginTest@26f3bbad]" name="loginTest" duration-ms="42511" started-at="2016-05-24T13:16:53Z" data-provider="OrderingDataProvider" finished-at="2016-05-24T13:17:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=24769_itv2_selenium, Password=qcontrol1, Runmode=Y, Expected_Result=FAILURE, Browser=Mozilla, Result=}]]>
              </value>
            </param>
          </params>
        </test-method> <!-- loginTest -->
      </class> <!-- com.ctliselenium.framework.datadriven.ordering.LoginTest -->
    </test> <!-- login Test -->
    <test name="Add Test" duration-ms="34817" started-at="2016-05-24T13:17:36Z" finished-at="2016-05-24T13:18:11Z">
      <class name="com.ctliselenium.framework.datadriven.ordering.AddTest">
        <test-method status="PASS" signature="initLogs()[pri:0, instance:com.ctliselenium.framework.datadriven.ordering.AddTest@1d51a1a1]" name="initLogs" is-config="true" duration-ms="15" started-at="2016-05-24T13:17:36Z" finished-at="2016-05-24T13:17:36Z">
        </test-method> <!-- initLogs -->
        <test-method status="PASS" signature="AddTest(java.util.Hashtable)[pri:0, instance:com.ctliselenium.framework.datadriven.ordering.AddTest@1d51a1a1]" name="AddTest" duration-ms="34739" started-at="2016-05-24T13:17:36Z" data-provider="OrderingDataProvider" finished-at="2016-05-24T13:18:11Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Col5=C53, Col4=C43, Col3=C33, Runmode=Y, PurchaseDate=25/5/2016, Browser=Mozilla, Result=}]]>
              </value>
            </param>
          </params>
        </test-method> <!-- AddTest -->
      </class> <!-- com.ctliselenium.framework.datadriven.ordering.AddTest -->
    </test> <!-- Add Test -->
  </suite> <!-- Ordering suite -->
  <suite name="My Project" duration-ms="0" started-at="2016-05-24T13:20:59Z" finished-at="2016-05-24T13:20:59Z">
    <groups>
    </groups>
  </suite> <!-- My Project -->
  <suite name="Billing Suite" duration-ms="49583" started-at="2016-05-24T13:18:16Z" finished-at="2016-05-24T13:19:06Z">
    <groups>
    </groups>
    <test name=" View Bill Test" duration-ms="49583" started-at="2016-05-24T13:18:16Z" finished-at="2016-05-24T13:19:06Z">
      <class name="com.ctliselenium.framework.datadriven.Billing.ViewBillTest">
        <test-method status="PASS" signature="initLogs()[pri:0, instance:com.ctliselenium.framework.datadriven.Billing.ViewBillTest@3932f2b2]" name="initLogs" is-config="true" duration-ms="2" started-at="2016-05-24T13:18:16Z" finished-at="2016-05-24T13:18:16Z">
        </test-method> <!-- initLogs -->
        <test-method status="PASS" signature="ViewBillTest(java.util.Hashtable)[pri:0, instance:com.ctliselenium.framework.datadriven.Billing.ViewBillTest@3932f2b2]" name="ViewBillTest" duration-ms="49506" started-at="2016-05-24T13:18:17Z" data-provider="BillingDataProvider" finished-at="2016-05-24T13:19:06Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Col5=C53, Col4=C43, Col3=C33, Col2=C22, Runmode=Y, Browser=Mozilla, Result=}]]>
              </value>
            </param>
          </params>
        </test-method> <!-- ViewBillTest -->
      </class> <!-- com.ctliselenium.framework.datadriven.Billing.ViewBillTest -->
    </test> <!--  View Bill Test -->
  </suite> <!-- Billing Suite -->
</testng-results>
